<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QDialog" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Blackjack</string>
  </property>
  <widget class="QLabel" name="P1Label1">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>150</width>
     <height>210</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="P1Label2">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>460</y>
     <width>150</width>
     <height>210</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="P1Label3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>460</y>
     <width>150</width>
     <height>210</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="P1Label4">
   <property name="geometry">
    <rect>
     <x>485</x>
     <y>460</y>
     <width>150</width>
     <height>210</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>4</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="P1Label5">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>460</y>
     <width>150</width>
     <height>210</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>5</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="P1Label6">
   <property name="geometry">
    <rect>
     <x>795</x>
     <y>460</y>
     <width>150</width>
     <height>210</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>6</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="P1Label7">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>460</y>
     <width>150</width>
     <height>210</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>7</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="P1Label8">
   <property name="geometry">
    <rect>
     <x>1105</x>
     <y>460</y>
     <width>150</width>
     <height>210</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>8</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="compLabel8">
   <property name="geometry">
    <rect>
     <x>1105</x>
     <y>10</y>
     <width>150</width>
     <height>210</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Deck">
   <property name="geometry">
    <rect>
     <x>565</x>
     <y>235</y>
     <width>150</width>
     <height>210</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Deck</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="compLabel1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>150</width>
     <height>210</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="compLabel2">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>10</y>
     <width>150</width>
     <height>210</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="compLabel3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>150</width>
     <height>210</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="compLabel4">
   <property name="geometry">
    <rect>
     <x>485</x>
     <y>10</y>
     <width>150</width>
     <height>210</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="compLabel5">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>10</y>
     <width>150</width>
     <height>210</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="compLabel6">
   <property name="geometry">
    <rect>
     <x>795</x>
     <y>10</y>
     <width>150</width>
     <height>210</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="compLabel7">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>10</y>
     <width>150</width>
     <height>210</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>680</y>
     <width>1281</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="hitButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Hit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="standButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Stand</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="rematchButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Rematch</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="quitButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Quit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="themeCombo">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>260</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>240</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Theme</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>310</y>
     <width>231</width>
     <height>122</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Luminari</family>
    </font>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Luminari</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Games Played:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Luminari</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Wins:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Luminari</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Losses:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <family>Luminari</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Draws:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="gameCountLabel">
        <property name="font">
         <font>
          <family>Luminari</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="winsLabel">
        <property name="font">
         <font>
          <family>Luminari</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lossesLabel">
        <property name="font">
         <font>
          <family>Luminari</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="drawsLabel">
        <property name="font">
         <font>
          <family>Luminari</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>304</width>
     <height>46</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="P1Name">
      <property name="font">
       <font>
        <family>Kokonor</family>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Player 1</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="P1Score">
      <property name="font">
       <font>
        <family>Gurmukhi MN</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>score</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>301</width>
     <height>46</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_4">
    <item row="0" column="0">
     <widget class="QLabel" name="dealerName">
      <property name="font">
       <font>
        <family>Kokonor</family>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Dealer</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="compScore">
      <property name="font">
       <font>
        <family>Gurmukhi MN</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>score</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="Deck2">
   <property name="geometry">
    <rect>
     <x>567</x>
     <y>235</y>
     <width>150</width>
     <height>210</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Deck</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Deck3">
   <property name="geometry">
    <rect>
     <x>569</x>
     <y>235</y>
     <width>150</width>
     <height>210</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Deck</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Deck4">
   <property name="geometry">
    <rect>
     <x>571</x>
     <y>235</y>
     <width>150</width>
     <height>210</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Deck</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Deck5">
   <property name="geometry">
    <rect>
     <x>573</x>
     <y>235</y>
     <width>150</width>
     <height>210</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Deck</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>hitButton</tabstop>
  <tabstop>standButton</tabstop>
  <tabstop>rematchButton</tabstop>
  <tabstop>quitButton</tabstop>
  <tabstop>themeCombo</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
